contrib/fparser/Makefile.orig

```diff
@@ -95,6 +95,7 @@ CPPFLAGS=$(FEATURE_FLAGS) -I$(LIBMESH_DIR)/include/libmesh
 CXXFLAGS=-Wall -W -pedantic -ansi $(OPTIMIZATION)
 #CXXFLAGS += -Wunreachable-code
 #CXXFLAGS += -std=c++0x
+CXXFLAGS += -std=c++11
```

Comment:
Does this makefile actually still get used?  I thought this was just what we renamed their Makefile before we put our own Makefile.am in place.

Reply:
Yes, I had to use it to regenerate fpoptimizer.cc